ai_services/language/python/languageBatchSamples.py

```diff
@@ -0,0 +1,62 @@
+import oci
+
```

Comment:
can you set the timeout to 30 seconds with a comment
oci.base_client.timeout=30 (please check out)
"Batch API consumes longer than 2 seconds depending on the document and batch size, so increasing the read time out to 30 seconds"

Reply:
I read default read time out is already 60 seconds so that should be fine ?

`
# Default timeout value(second)
DEFAULT_CONNECTION_TIMEOUT = 10.0
DEFAULT_READ_TIMEOUT = 60.0
`